.github/workflows/ci.yml

```diff
@@ -104,3 +104,32 @@ jobs:
         run: |
           tensorflow/lite/micro/tools/ci_build/test_makefile.sh
           tensorflow/lite/micro/tools/ci_build/test_x86.sh
+
+  label_cleanup:
```

Comment:
we are also using the label trigger for the xtensa workflow (and potentially others as well).

https://github.com/tensorflow/tflite-micro/blob/0afb62f9d1074df11c92f259a2eb1fc936009b96/.github/workflows/xtensa.yml

Should we move this label removing step to its own workflow file? Can we get into a race condition where the label removal happens before all the workflow that rely on ci:run are triggered?

Reply:
I did a little bit of testing and it appears that the triggering happens and queues before execution. Let me try to figure out how to do a few more tests to fully confirm that.

Reply:
ok, that would make sense. I'm happy to try it out and if we notice that things are not working as intended we can re-evaluate.

I would say let's still move this to its own workflow.yml

Reply:
Yeah, so found this [pull request breakdown](https://frontside.com/blog/2020-05-26-github-actions-pull_request/) and the workflows are triggered from an event message json payload (which you can see in the variable we are using in these if conditionals). We are changing the issue itself, not the event json message, so we are good.